### ISSUE TITLE
Question : Lux

### ISSUE THREAD

**Issue by orangelizard** [garbled1/homeassistant_ecowitt#86]
I was wondering should there be a LUX sensor. I have the Ecowit HP2551 weather station and it has a LUX reading on the connected display so thought it might be available in HA. It would be handy for the automated lighting in my house. Kind regards, Jason

**Comment by garbled1** [comment]
Yeah, the sensor is called "solar_radiation".  I have an HP3551, I think it's part of the anemometer, along with the UV sensor?  Can't remember offhand.

Go to configuration -> devices -> find the ecowitt, and then you should be able to see all of it's sensors in one place.

**Comment by orangelizard** [comment]
Thank you for that information, I hadn't realised I could use the solar_radiation sensor in the same way as I had become so transfixed on LUX as that is what I was using on another sensor in a jam jar on the roof which recently passed away in a thunder storm.

**Comment by garbled1** [comment]
Ha, exact same thing happened to my onewire lux sensor.  Yeah, the solar_radiation one reports in W/m^2, you could probably do some math to translate that to lux with a template sensor.

**Comment by dubbleeagle** [comment]
I would love to have a LUX-value sensor in the integration as well. The info is available in ecowitt, so would be awesome to get the value directly as a sensor!

**Comment by zacs** [comment]
> I would love to have a LUX-value sensor in the integration as well. The info is available in ecowitt, so would be awesome to get the value directly as a sensor!

Came to say the same! Just installed the sensor and was hoping for lux. I will look into the payload and see if it can be pulled without doing math. Maybe a PR coming.
